### PR TITLE
Corrections made

### PR DESCRIPTION
**Pull request by tanya21034**
Fixed bugs.

### CODE REVIEW

File: src/data/contributors/tanya21034.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (Abhishek-kumar09) [inline src/data/contributors/tanya21034.json:1]: Please, delete this file.

Comment (Abhishek-kumar09) [reply src/data/contributors/tanya21034.json:1]: Also delete sample.txt that you have created.

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]

File: src/data/contributors/tanya21034.json (hunk 1 above)
Comment (tanya21034) [reply src/data/contributors/tanya21034.json]: Please check and suggest if any other changes are to be made. Thanks!

**Comment by Abhishek-kumar09** [comment]
Thanks @tanya21034 for your PR. Welcome to the community!

**Pull request merged**